examples/go-client/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV GOPATH /go
 WORKDIR /go/src/twilio-go
 COPY . .
 
-RUN go get -u github.com/twilio/twilio-go@main
+RUN go get -u github.com/twilio/twilio-go@unmarshal
```

Comment:
Need to pass tests until https://github.com/twilio/twilio-go/pull/158 is merged. Will be reverted before merging.